src/commands/changes.ts

```diff
@@ -47,8 +47,17 @@ function getSelectionsLines(selections: vscode.Selection[]) {
   const lines: number[] = []
 
   for (const selection of selections) {
-    const startLine = selection.start.line,
-          endLine = selection.end.line
+    let startLine = selection.start.line,
+        endLine = selection.end.line
+
+    if (startLine !== endLine && selection.end.character === 0) {
+      // If the selection ends after a line break, do not consider the next line
+      // selected. This is because a selection has to end on the very first
+      // caret position of the next line in order to select the last line break.
+      // For example, `line.rangeIncludingLineBreak()` does this:
```

Comment:
nit: `vscode.TextLine.rangeIncludingLineBreak` (note that it's a property, not a function).

Reply:
Nice catch. Fixed!